lib/cortex/snippets/client/helper.rb

```diff
@@ -1,9 +1,11 @@
+require 'cortex/snippets/client/webpage'
```

Comment:
The dependency on `Webpage` in this file can go away, since the heavy lifting will be going on in `Client`.